styleguide/components/41-organisms/programme/_programme.scss

```diff
@@ -0,0 +1,194 @@
+.programme {
+  width: calc(100% + 2 * #{$gutter-width});
+  margin-right: -$gutter-width;
+  margin-left: -$gutter-width;
+
+  > .inner-box {
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+
+    &::before {
+      @include spot-image('calendar');
+
+      @include tablet {
+        right: 0;
+        left: 0;
+      }
+    }
+  }
+
+  ///
+  /// Programme slot.
+  ///
+  &-slot {
+    max-width: $bp-max-content;
+    margin: 0 auto;
+
+    &-title {
+      margin: 0 auto 1.6rem;
+      font-size: 1.3rem;
+      text-align: center;
+    }
+
+    > dd {
+      max-width: none;
+      padding: 0;
+    }
+
+    & + & {
+      margin-top: 2.8rem;
+    }
+  }
+
+  ///
+  /// Programme teaser.
+  ///
+  &-teaser {
+    @include theme('background-color', 'color-secondary');
```

Comment:
Just nitpicking here but I usually add a newline after includes for readability.